src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write code here
+'use strict';
```

Comment:
The 'use strict' directive is declared twice. You only need to declare it once at the beginning of the script.